common/src/dns.rs

```diff
@@ -0,0 +1,108 @@
+//! Seed resolution.
+use log::debug;
+use std::net::{SocketAddr, ToSocketAddrs};
+
+use crate::network::Network;
+
+/// DNS feature
+#[derive(Debug, Copy, Clone)]
+pub enum SeedFeature {
+    /// x1, Full Node
+    NODE_NETWORK,
+    /// x5, all include in x1 plus Node network
+    NODE_BLOOM,
+    /// x9, all included in x1 plus Node Witness
+    NODE_WITNESS,
+    /// xd (aka x13), all included in x9 and x5
+    FULL_NODE,
+}
```

Comment:
How about `struct SeedServices { flags: bitcoin::ServiceFlags }`

Reply:
I think, much better, thank to point me to [this feature](https://github.com/rust-bitcoin/rust-bitcoin/blob/e99177c4fba9ece1eb219e7980981c099bc6f2d5/src/network/constants.rs#L125) of rust-bitcoin

Reply:
I think, much better, thank to point me to this feature of [rust-bitcoin](https://github.com/rust-bitcoin/rust-bitcoin/blob/e99177c4fba9ece1eb219e7980981c099bc6f2d5/src/network/constants.rs#L125)